modelci/cli/modelhub.py

```diff
@@ -196,3 +196,76 @@ def detail(model_id: str = typer.Argument(..., help='Model ID')):
     with requests.get(f'{app_settings.api_v1_prefix}/model/{model_id}') as r:
         data = r.json()
         model_detailed_view(MLModel.parse_obj(data))
+
+
+@app.command('convert')
+def convert(
+        src_framework: str = typer.Argument(..., help='the prototype model framework to be converted'),
+        dst_framework: str = typer.Argument(..., help='the destination model framework after converting'),
+        model_path: str = typer.Argument(..., help='path of the model to be converted'
+                                                   'such as \'/home/model\' for keras savedmodel or'
+                                                   '\'/home/model/xxx.suffix\' for a file'),
+        save_path: str = typer.Argument(..., help='save path of the converted model'),
+        shape: str = typer.Option(None, '-s', '--shape', help='input shape of model, such as\'-shape [28,28,1]\''),
+        datatype: str = typer.Option(None, '-d', '--datatype', help='data type of model input, such as\'float32\'')
+):
+    import modelci.hub.converter.converter as cvt
+    import time
+    way = (src_framework, dst_framework)
+
+    if way == ('keras', 'onnx'):
+        import onnx
+        import tensorflow as tf
+        loaded = tf.saved_model.load(model_path)
+        localtime = time.strftime("%d%H%M%S", time.localtime())
+        save_path = Path(save_path+f'/keras_model_{str(localtime)}.onnx')
```

Comment:
we have a 'saving standard', check https://github.com/cap-ntu/ML-Model-CI/blob/master/modelci/hub/utils.py#L80

Reply:
def generate_path(model_name: str, task: Task, framework: Framework, engine: Engine,
                  version: Union[ModelVersion, str, int]):``
What dose this 'version' mean? Is it a str assigned by users for version control or a realistic version of the framework or engine of the model ?

Reply:
version of the model. Check TFS document, we follow their definition.